Wikipedia/UI-V5/WMFSelfSizingWaterfallCollectionViewLayout.m

```diff
@@ -0,0 +1,14 @@
+
+#import "WMFSelfSizingWaterfallCollectionViewLayout.h"
+
+@implementation WMFSelfSizingWaterfallCollectionViewLayout
+
+- (CGSize)collectionViewContentSize {
+    if ([self.collectionView numberOfSections] == 1 && [self.collectionView numberOfItemsInSection:0] == 0) {
```

Comment:
this _only_ happens when we have 1 section w/ 0 items? or does it happen when any section has 0 items?